platforms/documentation/docs/src/snippets/bestPractices/useVersionCatalogs/tests/useVersionCatalogs.sample.conf

```diff
@@ -0,0 +1,15 @@
+commands: [{
+    executable: gradle
+    execution-subdirectory: avoid-this
+    args: "dependencies"
+    expected-output-file: useVersionCatalogs.out
+    allow-additional-output: true
+    allow-disordered-output: true
```

Comment:
🤔  Disordered is usually only be necessary if there's a chance tasks might run in a different order.  If we only run a single task, this isn't needed.

Reply:
Bad copy-paste. ty